### ISSUE TITLE
DAO Error for returning too many tiles should be configurable or suppressible

### ISSUE THREAD

**Issue by RussTorres** [saalfeldlab/render#191]
https://github.com/saalfeldlab/render/blob/b06be441f3c78e1423c54bce20b291752c6d0773/render-ws/src/main/java/org/janelia/render/service/dao/RenderDao.java#L1856

The above line (from current master) raises an error when a query returns over 50k tiles from the database. We have started 
acquiring TEM montages on the order of 70k-100k tiles, so this becomes an issue for handling those with our current model.
While we can change the hardcoding, it would be nice to pass this limit with a configuration or in the api call.

**Comment by trautmane** [comment]
Agreed ... I hit the same issue last year and lazily just bumped up the limit ( see https://github.com/saalfeldlab/render/commit/d9b9ae049f7c285ec549db940642809d98a8a3a1 ).  I'll parameterize the limit and post back here when that solution is available.
